### ISSUE TITLE
enhance relational database example

### ISSUE THREAD

**Issue by xrotwang** [fmatter/pyradigms#4]
The example at https://github.com/fmatter/pyradigms/blob/master/scripts/paradigm_to_relational.ipynb could be fleshed out by
- adding a `Language_ID` column to the long format, thereby turning the data into a CLDF dataset (modulo some column renaming)
- using `pycldf.db.Database` to load the data into SQLite

**Comment by xrotwang** [comment]
@fmatter  Let me know if you'd like a PR for the above.

**Comment by fmatter** [comment]
Absolutely, that would be great! To make version control easier, I've paired the `.ipynb` file with a `.py` file using [jupytext](https://github.com/mwouts/jupytext).
